sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_operations/_operations.py

```diff
@@ -260,7 +260,7 @@ def build_document_translation_get_documents_status_request(  # pylint: disable=
 
 
 def build_document_translation_get_supported_formats_request(  # pylint: disable=name-too-long
-    *, type: Optional[Union[str, _enums.FileFormatType]] = None, **kwargs: Any
+    *, type: Optional[Union[str, _models.FileFormatType]] = None, **kwargs: Any
```

Comment:
This looks like an error in the generated code? I see FileFormatType in _enums.py

Reply:
Ah nevermind I just saw the import...resolving